src/chronometer-centiseconds.js

```diff
@@ -1,37 +1,55 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+
+      if(printTimeCallback){
+        return printTimeCallback ();
+      }  
+    }, 10)
+    
   }
 
   getMinutes() {
-    // ... your code goes here
+    let minutes = Math.floor(this.currentTime / 6000);
+    return minutes;
   }
 
   getSeconds() {
-    // ... your code goes here
+    let seconds = Math.floor(this.currentTime / 100) % 60;
+    return seconds;
   }
 
   getCentiseconds() {
-    // ... your code goes here
+    let centiseconds = Math.floor(this.currentTime % 100);
+    return centiseconds;
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    if(value < 10) {
```

Comment:
This does work fine, and there's another way to do this using string interpolation, like:

```
if (value < 10) { 
      return `0${value}`;
    } else {
      return `${value}`;
    }
```